CHANGELOG.md

```diff
@@ -9,13 +9,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
+### Fixed
+
+## [3.4.1] 2019-02-12
+*Released to Fire TV 4K.*
+### Added
```

Comment:
No need to leave these blank: you can remove these sections if they're unused. This and "Changed". FYI: there are actually three more sections we don't seem to use often: "Deprecated", "Security", and "Removed".